packages/yarnpkg-core/sources/TelemetryManager.ts

```diff
@@ -85,7 +86,7 @@ export class TelemetryManager {
   }
 
   private reportEnumerator(metric: MetricName, value: string) {
-    miscUtils.getSetWithDefault(this.enumerators, metric).add(value);
+    miscUtils.getSetWithDefault(this.enumerators, metric).add(hashUtils.makeHash(value));
```

Comment:
Should we update the old values as well so that existing values don't count twice?

Reply:
It would add a fair bit of complexity depending on how backward compatible we want it to be, I'm not sure it's needed 🤔 Plus it won't count things twice for very long anyway (since that'll only happen the week a project gets upgraded to 3.x).